ui/nuxeo-filters-behavior.js

```diff
@@ -250,4 +251,26 @@ export const FiltersBehavior = {
   hasAdministrationPermissions(user) {
     return user && (user.isAdministrator || this.isMember(user, 'powerusers'));
   },
+
+  /**
+   * Checks if the given document has running workflows
```

Comment:
since I've been working on JSDocs recently, I would suggest to also add the param and return tags on both functions :)

Reply:
Thanks Andrea, good point, but as this another repo than our aspera one. I just kept the things as we did on the whole file. no js doc with the required tags.  I think if want to do it we should create a separate ticket and do on element repo.

wdyt?

Reply:
indeed! we can also talk to Beltran and see if we open a ticket for it.